src/app/layout.tsx

```diff
@@ -14,10 +18,15 @@ export const metadata: Metadata = {
 export default function RootLayout({
   children,
 }: Readonly<{ children: React.ReactNode }>) {
+
   return (
     <html lang="en" className={`${GeistSans.variable}`}>
-      <body className="relative scrollbar-thin scroll-smooth">
+      <body className="relative scrollbar-thin scroll-smooth flex min-h-screen flex-col items-center justify-center">
```

Comment:
I checked this `smooth-scroll` does not work. Is it?
Also `scrollbar-thin` is not working.

Reply:
Probably need to add this `class` inside `<html>` classname